pom.xml

```diff
@@ -620,50 +684,19 @@
                 <goal>jar</goal>
               </goals>
             </execution>
-            <execution>
-              <id>default-test-jar</id>
```

Comment:
I think we need to keep this in the default profile since we have modules that depend on others modules test jars because of re-used testing utilities. Will need to factor those out to separate testing packages.